dace/frontend/python/newast.py

```diff
@@ -3141,7 +3183,13 @@ def visit_AnnAssign(self, node: ast.AnnAssign):
             dtype = None
             warnings.warn('typeclass {} is not supported'.format(type_name))
         if node.value is None and dtype is not None:  # Annotating type without assignment
-            self.annotated_types[rname(node.target)] = dtype
+            try:
+                array_type = eval(astutils.unparse(node.annotation), globals(),
```

Comment:
should this maybe be `self.globals`?

Reply:
Hmm, could be I guess. In any case, this is supposed to work for array types, i.e. `dace.whatever[shape]`, so both should work (both have dace).

Reply:
Fixed.